features/steps/notification_service_dependencies.py

```diff
@@ -43,6 +43,21 @@ def check_service_log_availability(context, host, port):
     assert x.status_code == 200, "service log is not up"
 
 
+@given("token refreshment server is available on {host:w}:{port:d}")
+def check_token_refreshment_availability(context, host, port):
+    """Check if token refreshment server is available at given address"""
+    if not str(host).startswith("http"):
```

Comment:
wont' work in https case, prob. change "http" -> "http:" or whatever

Reply:
I think the same as Dan, it should be fine with `https` since it has `http` as prefix

Reply:
yup, I thought about having the check for a protocol, not just for a string prefix. OTOH the probabiity that someone's machine name starts with "http" is a bit low

TODO for myself: test our services on such machine!

Reply:
So maybe I add both `http:` and h`ttps:` prefixes as the options? (also it should be changed in other check functions in this file)

Reply:
Will do it in next PR together with changing other checks.